handler/instance-started.go

```diff
@@ -112,30 +115,39 @@ func (h *InstanceStarted) getCodeListsFromInstance(i *recipe.Instance) ([]string
 
 	var codelists []string
 	for _, cl := range i.CodeLists{
-		codelists = append(codelists, cl.Name)
+		codelists = append(codelists, cl.ID)
 	}
 
 	return codelists, nil
 }
 
-func (h *InstanceStarted) createInstanceRequest(cb cantabular.Codebook) (*dataset.UpdateInstance, error){
+func (h *InstanceStarted) createUpdateInstanceRequest(cb cantabular.Codebook, e *event.InstanceStarted, clURL string) dataset.UpdateInstance{
 	req := dataset.UpdateInstance{
 		Edition: "2021",
 		CSVHeader: []string{"ftb_table"},
+		InstanceID: e.InstanceID,
```

Comment:
Do we need to set the state to `submitted`? 
```
		State:      dataset.StateSubmitted.String()
```

Reply:
It seems to happen automatically, perhaps calling PutInstance.. but looking at the instance mongo you can see `"state": "submitted"`